src/display/display_utils.js

```diff
@@ -770,7 +770,41 @@ const SupportedImageMimeTypes = [
   "image/x-icon",
 ];
 
+function changeLightness(r, g, b, lumCallback) {
```

Comment:
The `lumCallback` value given in the callsites seems to be equal everywhere, so can we perhaps make that value a default argument to simplify the callers? If we don't foresee changing this we could also just inline that function here.